x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameTransformProgressIT.java

```diff
@@ -163,6 +161,23 @@ public void testGetProgress() throws Exception {
         assertThat(progress.getRemainingDocs(), equalTo(35L));
         assertThat(progress.getPercentComplete(), equalTo(0.0));
 
+        histgramGroupConfig = new GroupConfig(Collections.emptyMap(),
```

Comment:
Does this method cover multiple independent test case scenarios?
How hard would it be to split it into a few independent test cases?

Reply:
@przemekwitek not too difficult, but the concern is creating and destroying indices, and speed.